cypress/integration/accessibility_Upsell_spec.js

```diff
@@ -15,4 +15,15 @@ describe('Upsell Accessibility check', () => {
     });
     cy.checkA11y();
   });
+
+  // Keep disabled until converting to new Code editor
+  cy.configureAxe({
+    rules: [
+      {
+        id: 'color-contrast',
+        enabled: false,
+      },
+    ],
+  });
+  cy.checkA11y();
```

Comment:
instead of adding this here, you'll want to edit lines 7-17 above to look like: 
```

it('Tests accessibility on the Upsell page', () => {
    cy.configureAxe({
      rules: [
        {
          id: 'scrollable-region-focusable',
          enabled: false,
        },
{
          id: 'color-contrast',
          enabled: false,
        },
      ],
    });
    cy.checkA11y();
  });
```

Reply:
(with better spacing :P )

Reply:
Done!